utils/entrypoint.sh

```diff
@@ -184,7 +184,7 @@ start_mysql () {
 
     if [ "$(mysql_datadir_exists)" -eq "0" ]; then
         echo " * First run of MYSQL, initializing DB."
-        mysql_install_db --user=mysql --datadir=/var/lib/mysql/ > /dev/null 2>&1
+        /usr/sbin/mysqld --initialize-insecure --user=mysql --datadir=/var/lib/mysql/ > /dev/null 2>&1
```

Comment:
This will only work on Debian based distros. We can't use hard coded paths and we need to use mysqld_safe instead of mysqld. I'll merge the changes in a different commit instead of this pr.

Reply:
i'm agree with you.